tools/virt-builder/about.md

```diff
@@ -47,38 +47,45 @@ root password, only the bare minimum installed software, etc.).
 
 Import the above disk image into libvirt:
 
-    $ virt-install --import \
-       --name f23vm1 --ram 2048 \
-       --disk path=fedora-23.img,format=raw --os-variant fedora23
+    $ virt-install \
+       --name f27vm1 --ram 2048 \
+       --disk path=fedora-27.img,format=raw \
+       --os-variant fedora26 \
+       --import \
+
+Note: The `--os-variant` doesn't _strictly_ have to be F27; the nearest
+possible available Fedora variant is fine.  To find the list of variants
+for your current Fedora release, run: `osinfo-query os | grep fedora`.
 
 ## Examples to create custom virtual machines
 
-###  Create a Fedora 23 VM and update it
+###  Create an up-to-date Fedora 27 VM
 
-Prepare a QCOW2 Fedora 23 VM, with 40GB disk size, and update to latest
+Prepare a QCOW2 Fedora 27 VM, with 40GB disk size, and update to latest
 available packages:
 
-    $ virt-builder fedora-23 -o f23vm2.qcow2 --format qcow2 \
+    $ virt-builder fedora-27 -o f27vm2.qcow2 --format qcow2 \
         --update --selinux-relabel --size 40G
 
 Import the disk image into libvirt, and provide it 4GB of memory:
 
-    $ virt-install --name f23vm2 --ram 4096 \
-        --disk path=f23vm2.qcow2,format=qcow2,cache=writeback \
-        --nographics --import --os-variant fedora23
+    $ virt-install --name f27vm2 --ram 4096 \
+        --disk path=f27vm2.qcow2,format=qcow2,cache=writeback \
+        --nographics --os-variant fedora26 --import
```

Comment:
Same here, please also fix `fedora26` -> `fedora27`.

Reply:
It was intentional (but probably to avoid confusion, I'll just update it to `fedora27`).  Because see the note about `--os-variant` : on F27 machines, you don't have the `fedora26` variant when you run `osinfo-query os | grep fedora` (yes, I've reported it to upstream 'libosinfo').  Also it is not mandatory to provide `fedora27`, as you can even provide `fedora25` --- i.e. the nearest possible available Fedora variant is fine.